website/assets/styles/pages/get-started.less

```diff
@@ -41,29 +43,34 @@
     color: #192147;
     font-weight: bold;
     user-select: none;
-    transition: 0.2s linear;
-    -o-transition: 0.2s linear;
-    -ms-transition: 0.2s linear;
-    -moz-transition: 0.2s linear;
-    -webkit-transition: 0.2s linear;
+    transition: 0.2s ease-in-out;
+    -o-transition: 0.2s ease-in-out;
+    -ms-transition: 0.2s ease-in-out;
+    -moz-transition: 0.2s ease-in-out;
+    -webkit-transition: 0.2s ease-in-out;
   }
 
   #arrow:before {
-    content: url('/images/arrow-right-16x16@2x.png');
+    content: url("/images/arrow-right-16x16@2x.png");
```

Comment:
Any particular reason for the change to double-quotes here? 
(If not, might be good to make sure you're sticking with the conventions that are being used in other CSS files. If this was an intentional change that y'all already talked about, ignore me!)

Reply:
I think this was probably an auto format change.  I'll revert to the single quote

Reply:
if you find an editor plugin that respects .lesshintrc files, and you open your editor from within the `website/` dir, it might just work™